mlflow/entities/span_status.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from opentelemetry import trace as trace_api
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from mlflow.tracing.types.constant import TraceStatusCode
+
+
+@dataclass
+class SpanStatus:
+    """
+    Status of the span or the trace.
+
+    Args:
+        status_code: The status code of the span or the trace.
+        description: Description of the status. Optional.
+    """
+
+    status_code: TraceStatusCode
+    description: str = ""
+
+    def to_otel_status(self) -> trace_api.Status:
+        """Convert our status object to OpenTelemetry status object."""
+        if self.status_code == TraceStatusCode.OK:
```

Comment:
maybe we make a map as the class' member?

Reply:
oh I thought we can't add class variable for dataclass (without including it in serialized form), but it seems we can do with ClassVar annotation. That sounds cleaner then!